konnectivity-client/pkg/client/conn.go

```diff
@@ -54,7 +54,7 @@ func (c *conn) Write(data []byte) (n int, err error) {
 		},
 	}
 
-	klog.Infof("[tracing] send req %+v", req)
+	klog.Infof("[tracing] send req %+v", req.Type)
```

Comment:
Should probably set the v level to 5 or higher. Even without the data load this is going to be VERY chatty.

Reply:
Can you add a v(6) log for the data of req, and print it as %s?

Reply:
ack I can change to v(6). Does printing as %s change anything? It'll still look incomprehensible since it's just a sequence of bytes?

Reply:
My thought is that the data is encoded in either proto or json, where proto has some bits readable, and json is entirely readable.

Reply:
Try it to be sure. I believe as a %s, PacketType.String() will be called on req.Type which should translate to something like DIAL_REQ or DATA.

Reply:
https://play.golang.org/p/h8yTwUATaUw

Reply:
Please ignore my comment about logging the request data. It might lead to accidentally logging secrets.

Reply:
Thanks for the comments. I changed it to

`klog.V(6).Infof("[tracing] send req %s", req.Type)`